src-vue/src/utils/thunderstore/ThunderstoreModVersion.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface ThunderstoreModVersion {
+    description: string;
+    icon: string;
+    version_number: string;
+    download_url: string;
+    downloads: number;
+    date_created: string;
+}
```

Comment:
This should also export `full_name: string` as that field contains the mod string needed to be passed to the backend for mod install. ^^

![image](https://user-images.githubusercontent.com/40122905/202733691-eb32fd78-8ba5-423e-85a0-cf683b3c4414.png)

Reply:
Added in b8ef66662cef9d5bc58b26a07af418024c3f8f9e